.gitignore

```diff
@@ -1,5 +1,6 @@
 target
 .sbtopts
 project/.sbt
+.idea
```

Comment:
please follow instruction from this stackoverflow post

Reply:
fixed